appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -319,33 +318,14 @@ static ControllerLinkBuilder getConfirmMobileNumberLinkBuilder(UUID userId)
 		return linkTo(methodOn.confirmMobileNumber(Optional.empty(), userId, null));
 	}
 
-	private HttpEntity<UserResource> addUser(Optional<String> password, Optional<String> overwriteUserConfirmationCode,
-			UserDto user)
+	private HttpEntity<UserResource> addUser(Optional<String> overwriteUserConfirmationCode, UserDto user)
 	{
-		checkAppProvidedPassword(password);
-		SecretKey secretKey = password.map(p -> CryptoSession.getSecretKey(password.get()))
-				.orElse(SecretKeyUtil.generateRandomSecretKey());
-		try (CryptoSession cryptoSession = CryptoSession.start(secretKey))
+		try (CryptoSession cryptoSession = CryptoSession.start(SecretKeyUtil.generateRandomSecretKey()))
 		{
 			return createResponse(userService.addUser(user, overwriteUserConfirmationCode), true, HttpStatus.CREATED);
 		}
 	}
 
-	private void checkAppProvidedPassword(Optional<String> password)
-	{
-		if (password.isPresent())
-		{
-			if (yonaProperties.getSecurity().isAppProvidedPasswordEnabled())
-			{
-				logger.warn("Creating user with app-provided password");
-			}
-			else
-			{
-				throw InvalidDataException.appProvidedPasswordNotSupported();
```

Comment:
No, it hasn't, and it shouldn't, as it's still in use at line 229 (``updateUserCreatedOnBuddyRequest``). Honestly, I forgot to remove it, but when I tried to remove it, in follow up to your comment, I see it's still in use.